src/main/webapp/app/controllers/submission/adminSubmissionViewController.js

```diff
@@ -246,13 +246,24 @@ vireo.controller("AdminSubmissionViewController", function ($anchorScroll, $cont
             var documentTypePredicates = [];
             for (var i in $scope.submission.fieldValues) {
                 var fieldValue = $scope.submission.fieldValues[i];
-                if (fieldValue.fieldPredicate.documentTypePredicate) {
+                if (fieldValue.fieldPredicate.documentTypePredicate && !containsPredicate(documentTypePredicates, fieldValue.fieldPredicate)) {
                     documentTypePredicates.push(fieldValue.fieldPredicate);
                 }
             }
             return documentTypePredicates;
         };
 
+        var containsPredicate = function (array, predicate) {
```

Comment:
This can be simplified:

```return array.map(function(p) { return p.id; }).indexOf(predicate.id) >= 0```

Reply:
Thanks, I've updated the PR to include this.